sktime/utils/check_imports.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+from importlib import import_module
+
+
+def _check_imports(package):
+    try:
+        import_module(package)
+    except ModuleNotFoundError as e:
+        raise Exception(
```

Comment:
```suggestion
        raise ModuleNotFoundError(
```
I'd prefer to raise a more specific exception and adjust the unit tests correspondingly.